docs/ops.html

```diff
@@ -1366,8 +1366,9 @@ <h4 class="anchor-heading"><a id="replace_disk" class="anchor-link"></a><a href=
   <p>If the data in the cluster metadata directory is lost either because of hardware failure or the hardware needs to be replaced, care should be taken when provisioning the new controller node. The new controller node should not be formatted and started until the majority of the controllers have all of the committed data. To determine if the majority of the controllers have the committed data, run the kafka-metadata-quorum.sh tool to describe the replication status:</p>
 
   <pre><code class="language-bash">$ bin/kafka-metadata-quorum.sh --bootstrap-server localhost:9092 describe --replication
-NodeId  LogEndOffset    Lag     LastFetchTimestamp      LastCaughtUpTimestamp   Status
-1       25806           0       1662500992757           1662500992757           Leader
+NodeId	DirectoryId           	LogEndOffset	Lag	LastFetchTimestamp	LastCaughtUpTimestamp	Status
```

Comment:
LGTM, and I find that DirectoryIds in out `MetadataQuorumCommandTest` are always "AAAAAAAA...", we may consider improving out test kit.

Reply:
Interesting issue... I saw what you mentioned from `testDescribeQuorumReplicationSuccessful`. The DirectoryId is always `AAAAAAAAAAAAAAAAAAAAAA` even though they are from different directory. And the string `AAAAAAAAAAAAAAAAAAAAAA` equals to `UUID.zero`. I'm still trying to find out the root cause. Will file a PR later. Thanks for bring this up !

Reply:
The reason MetadataQuorumCommandTest output DirectoryId `AAAAAAAAAAAAAAAAAAAAAA` is that we still use `kraft.version = 0` in `RaftClusterInvocationContext`, simply modify kraft.version to 1 could resolve this issue. And https://issues.apache.org/jira/browse/KAFKA-16653 will address that.